controller/backup_volume_controller.go

```diff
@@ -284,11 +284,27 @@ func (bvc *BackupVolumeController) reconcile(backupVolumeName string) (err error
 		return err
 	}
 
+	// Get `Time to live` after failed backup was marked as `Error` or `Unknown`,
+	// default: 180s and it is shorter than `Backupstore Poll Interval` by default, so failed backups should be deleted once the backup volume is updated.
+	failedBackupTTL, err := bvc.ds.GetSettingAsInt(types.SettingNameFailedBackupTTL)
```

Comment:
This means, the actual TTL is controlled by the poll interval. Or more precisely, it's the multiple of the poll interval, right?

Reply:
Yes, `failed backup TTL` is more like `at least` period the longhorn would keep the failed backup.

Reply:
Then it's better to explain more in the description part.

Reply:
Modified as David's Suggestion. Thanks.